harvdev_utils/chado_functions/feature.py

```diff
@@ -280,7 +280,7 @@ def feature_synonym_lookup(session, type_name, synonym_name, organism_id=None, c
     raise DataError("DataError: Could not find current unique synonym '{}', sgml = '{}' for type '{}'.".format(synonym_name, synonym_sgml, cvterm_name))
 
 
-def feature_symbol_lookup(session, type_name, synonym_name, organism_id=None, cv_name='synonym type', cvterm_name='symbol'):
+def feature_symbol_lookup(session, type_name, synonym_name, organism_id=None, cv_name='synonym type', cvterm_name='symbol', check_unique=True):
```

Comment:
Need to add "check_unique" description to the "Args" section of the docstring.

Reply:
Ah yes of course.